site/src/util/workspace.ts

```diff
@@ -186,6 +186,37 @@ export const getDisplayWorkspaceBuildStatus = (
   }
 }
 
+export const DisplayWorkspaceBuildInitiatedByLanguage = {
+  autostart: "system/autostart",
+  autostop: "system/autostop",
+}
+
+export const getDisplayWorkspaceBuildInitiatedBy = (
+  theme: Theme,
+  build: TypesGen.WorkspaceBuild,
+): {
+  color: string
+  initiatedBy: string
+} => {
+  switch (build.reason) {
+    case "initiator":
+      return {
+        color: theme.palette.text.secondary,
+        initiatedBy: build.initiator_name,
+      }
+    case "autostart":
+      return {
+        color: theme.palette.secondary.dark,
+        initiatedBy: DisplayWorkspaceBuildInitiatedByLanguage.autostart,
+      }
+    case "autostop":
+      return {
+        color: theme.palette.secondary.dark,
+        initiatedBy: DisplayWorkspaceBuildInitiatedByLanguage.autostop,
+      }
```

Comment:
My feeling (and I know this is boring of me) is that we probably don't need to highlight these values in purple. I am concerned we will be drawing too much attention to information that, while important, is not particularly actionable. That being said, I love that you tried a different design! Maybe post this in the dev-ux channel to get more opinions - I'm not a designer by any stretch of the imagination haha.